diff --git a/node_modules/@sgtpooki/react-app-rewired-esm/config-overrides.js b/node_modules/@sgtpooki/react-app-rewired-esm/config-overrides.js
index 6791f7c..2a7255f 100644
--- a/node_modules/@sgtpooki/react-app-rewired-esm/config-overrides.js
+++ b/node_modules/@sgtpooki/react-app-rewired-esm/config-overrides.js
@@ -1,4 +1,5 @@
 import { createRequire } from "node:module";
+import { pathToFileURL } from "node:url";
 export const require = createRequire(import.meta.url);
 
 const { paths } = require("./index.cjs");
@@ -6,7 +7,7 @@ const { paths } = require("./index.cjs");
 // before overrides scripts are read
 require(paths.scriptVersion + "/config/env");
 
-let override = await import(paths.configOverrides + ".js");
+let override = await import(pathToFileURL(paths.configOverrides + ".js").href);
 override = override.default || override;
 
 const webpack =
