diff --git a/node_modules/ipld-explorer-components/dist/providers/helia.js b/node_modules/ipld-explorer-components/dist/providers/helia.js
index ed6e5c5..849ef70 100644
--- a/node_modules/ipld-explorer-components/dist/providers/helia.js
+++ b/node_modules/ipld-explorer-components/dist/providers/helia.js
@@ -42,17 +42,18 @@ const HeliaProvider = ({ children }) => {
     const heliaHttpVersion = packageJson.dependencies["@helia/http"];
     return `@helia/http@${heliaHttpVersion}`;
   };
-  const doInitHelia = useCallback(async () => {
+  const doInitHelia = useCallback(async (kuboGatewayOptions) => {
     dispatch({ type: "HELIA_INIT_STARTED" });
     try {
-      const helia2 = await initHelia(state.kuboGatewayOptions);
+      const helia2 = await initHelia({...state.kuboGatewayOptions, ...kuboGatewayOptions });
       setHelia(helia2);
-      dispatch({ type: "HELIA_INIT_FINISHED", payload: { kuboGatewayOptions: state.kuboGatewayOptions } });
+      dispatch({ type: "HELIA_INIT_FINISHED", payload: { kuboGatewayOptions: {...state.kuboGatewayOptions, ...kuboGatewayOptions } } });
     } catch (error) {
       console.error("doInitHelia error", error);
       dispatch({ type: "HELIA_INIT_FAILED", error });
     }
   }, [state.kuboGatewayOptions, setHelia, dispatch]);
+
   useEffect(() => {
     void doInitHelia();
   }, []);
