diff --git a/node_modules/istextorbinary/edition-browsers/index.js b/node_modules/istextorbinary/edition-browsers/index.js
index 34af306..53cceee 100644
--- a/node_modules/istextorbinary/edition-browsers/index.js
+++ b/node_modules/istextorbinary/edition-browsers/index.js
@@ -1,5 +1,5 @@
 /* eslint no-use-before-define:0 */
-import * as pathUtil from 'path';
+import * as pathUtil from 'path-browserify';
 import textExtensions from 'textextensions';
 import binaryExtensions from 'binaryextensions';
 /**
