diff --git a/node_modules/multiaddr/src/convert.js b/node_modules/multiaddr/src/convert.js
index 7258b50..09f28f9 100644
--- a/node_modules/multiaddr/src/convert.js
+++ b/node_modules/multiaddr/src/convert.js
@@ -39,6 +39,7 @@ Convert.toString = function convertToString (proto, buf) {
     case 56: // dnsaddr
     case 400: // unix
     case 777: // memory
+    case 466: // certhash
       return bytes2str(buf)
 
     case 421: // ipfs
@@ -72,6 +73,7 @@ Convert.toBytes = function convertToBytes (proto, str) {
     case 56: // dnsaddr
     case 400: // unix
     case 777: // memory
+    case 466: // certhash
       return str2bytes(str)
 
     case 421: // ipfs
diff --git a/node_modules/multiaddr/src/protocols-table.js b/node_modules/multiaddr/src/protocols-table.js
index 4578e43..db46337 100644
--- a/node_modules/multiaddr/src/protocols-table.js
+++ b/node_modules/multiaddr/src/protocols-table.js
@@ -52,6 +52,9 @@ Protocols.table = [
   [445, 296, 'onion3'],
   [446, V, 'garlic64'],
   [460, 0, 'quic'],
+  [461, 0, 'quic-v1'],
+  [465, 0, 'webtransport'],
+  [466, V, 'certhash'],
   [477, 0, 'ws'],
   [478, 0, 'wss'],
   [479, 0, 'p2p-websocket-star'],
